src/types/student/student-entity.ts

```diff
@@ -0,0 +1,43 @@
+import { StudentStatus, TypeWork, ContractType } from './enums';
+
+export interface StudentEntity {
```

Comment:
Ty bym się zastanowił żeby zrobić 2 osobne interfejsy i potem zrobić jeszcze jeden który by zbierał dane z obu

Reply:
ja myślę, że tak chyba lepiej, a skoro ma działać tak samo to może niech już tak zostanie. Chyba, że pod jakimiś względami będzie to miało znaczenie?

Reply:
W tej chwili nie mamy typu dla tych 6 pól które są omitowane w studentProfile a wydaje mi się że gdzieś taki typ będzie potrzebny

Reply:
w sumie racja... może być coś takiego przy jakiejś inicjacji profilu kursanta, zanim zostanie stworzony jego profil, teraz czaję o co chodzi, zaraz pomyślę nad tym

Reply:
można zrobić dwa osobne interfejsy i trzeci który będzie implementował te dwa

Reply:
dobra, zrobione, mam nadzieję, że niczego tam nie pogubiłem po drodze drodze :D